### DISCUSSION TITLE
Youtube Music вообще отказывается нормально работать

### DISCUSSION THREAD

**Discussion by Royynn** [dovecoteescapee/ByeDPIAndroid#202] [Q&A] [answered]
москва, провайдеры мобильной сети мтс, т2, домашний мгтс. перепробовал уже почти все конфиги, иногда подгружается но через секунд 10 полный провал. впн с протоколом vless тоже не помогает.
если кто уже решил эту проблему то помогите плиз

**Comment by m7868379** [comment]
та же самая история, не работает и с перебором конфигов; не так давно ещё работало - проблема наблюдается недавно и исключительно в ютуб мьюзик, сам ютуб работает

**Reply by Royynn** [reply, marked as answer]
> та же самая история, не работает и с перебором конфигов; не так давно ещё работало - проблема наблюдается недавно и исключительно в ютуб мьюзик, сам ютуб работает

я разобрался с проблемой, проблема вообще не в байдпиай, гугл гасят Revanced мод.
скачай самую последнюю версию YTM Revanced Extended by inoita00 на 4pda и в настройках включи spoof streaming data.
